Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageRightHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageRightHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+
+        // Left-hand pin
+        if let image =  UIImage(named: "ImageCalloutLeftHanded") {
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageLeftHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageLeftHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+    }
+
+    private func addFeatures() -> FeatureCollection {
+
+        var features = [Turf.Feature]()
+
+        let featureList = [
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.714203, -74.006314), highlighted: false, sortOrder: 0, tailPosition: .left, label: "Chambers & Broadway - Lefthand Stem", imageName: "imageLeftHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.707918, -74.006008), highlighted: false, sortOrder: 0, tailPosition: .right, label: "Cliff & John - Righthand Stem", imageName: "imageRightHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.716281, -74.004526), highlighted: true, sortOrder: 1, tailPosition: .right, label: "Broadway & Worth - Right Highlighted", imageName: "imageRightHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.710194, -74.004248), highlighted: true, sortOrder: 1, tailPosition: .left, label: "Spruce & Gold - Left Highlighted", imageName: "imageLeftHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.7128, -74.0060), highlighted: true, sortOrder: 2, tailPosition: .center, label: "City Hall - Centered Highlighted", imageName: "imageCentered-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.711427, -74.008614), highlighted: false, sortOrder: 3, tailPosition: .center, label: "Broadway & Vesey - Centered Stem", imageName: "imageCentered")
+        ]
```

Comment:
Calling the lower array featureList is confusing given that now we have Turf.Feature and DebugFeature. Maybe we could rename DebugFeature in terms of just some generic data entity and avoid using the term feature in its name